src/main/java/org/genomicsdb/model/ImportConfig.java

```diff
@@ -69,7 +71,8 @@ public ImportConfig(final GenomicsDBImportConfiguration.ImportConfiguration impo
                         final Set<VCFHeaderLine> mergedHeader,
                         final Map<String, URI> sampleNameToVcfPath,
                         final Func<Map<String, URI>, Integer, Integer,
-                                Map<String, FeatureReader<VariantContext>>> sampleToReaderMapCreator) {
+                                Map<String, FeatureReader<VariantContext>>> sampleToReaderMapCreator,
```

Comment:
Fix indentation.

Reply:
Did you want the Map intend reduced by two spaces? To clarify, the Map<String, FeatureReader> is part of the Func argument to the constructor there....